src/hooks/login/useExtensionLogin.ts

```diff
@@ -23,13 +22,14 @@ export type UseExtensionLoginReturnType = [
 export const useExtensionLogin = ({
   callbackRoute,
   token,
-  redirectAfterLogin = false
 }: UseExtensionLoginPropsType): UseExtensionLoginReturnType => {
   const [error, setError] = useState('');
   const [isLoading, setIsLoading] = useState(false);
   const isLoggedIn = useSelector(isLoggedInSelector);
   const dispatch = useDispatch();
 
+  const redirectAfterLogin = Boolean(callbackRoute);
```

Comment:
Maybe we should just move this logic inside optionalRedirect, to make sure there is no confusion